### PR TITLE
Removes ignore handleCSTCallback

### PR DESCRIPTION
**Pull request by emmberk**
because it was fixed by https://github.com/connexta/multi-int-store/pull/48

### CODE REVIEW

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
The build failed due to a know issue downloading CVEs. Itests succeeded. I'm overriding the CI build on this PR.

File: multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java [hunk 1]
```diff
@@ -53,8 +53,6 @@ public void stop() {
   @Test
   public void testContextLoads() {}
 
-  // TODO: Does not pass because it did not talk to Solr during integration test.
-  //  "IOException occurred when talking to server at: http://localhost:32773/solr"
   @Ignore
```
Comment (vinamartin) [inline multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java]: Is this to remove the ignore or just the comment?

Comment (jhunzik) [inline multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java]: ❗️ This can be removed too right?

Comment (emmberk) [reply multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java]: 🤦‍♀

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
`./gradlew clean build` succeeded for me locally.

**Pull request merged**